src/env.d.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+/**
+ * @description - Typescript Intellisense for VITE Environmental variables
+ * Add VITE local variables here
+ * Add VITE local variables here
+ */
+interface ImportMetaEnv {
+  /**
+   * Notex hosted backend server url from environmental variable file(.env)
+   */
+  VITE_API_URL: string;
+  /**
+   * Notex google oauth url from environmental variable file (.env )
+   */
+  VITE_GOOGLE_OAUTH_URL: string;
+  /**
+   * local backend server url from environmental variable file (.env)
+   */
+  VITE_API_LOCAL_URL: string;
```

Comment:
we don't have such env variable

Reply:
its has being added

Reply:
why? PR description says nothing about it. The reason of this vars is unclear, since we already have them. They are not used in a code.

Reply:
I will comment those out, probably for my use alone